tests/test_decorators.py

```diff
@@ -106,12 +223,14 @@ def test_success_is_failure_tree():
     py_trees.display.print_ascii_tree(root)
     visitor = py_trees.visitors.DebugVisitor()
     py_trees.tests.tick_tree(root, 1, 1, visitor, print_snapshot=True)
- 
+
     print("\n--------- Assertions ---------\n")
     print("failure.status == py_trees.common.Status.FAILURE")
     assert(failure.status == py_trees.common.Status.FAILURE)
     print("going_down.status == py_trees.common.Status.FAILURE")
     assert(going_down.status == py_trees.common.Status.FAILURE)
+    print("going_down.decorated.status == py_trees.common.Status.SUCCESS")
```

Comment:
Originally planned to leave these out since I wanted to make the tests implementation independent, but yes, they are worth just covering. Added them in the followup PR.